libp2p/protocols/secure/secio.nim

```diff
@@ -309,7 +310,7 @@ method handshake*(s: Secio, conn: Connection, initiator: bool = false): Future[S
   trace "Local proposal", schemes = SecioExchanges,
                           ciphers = SecioCiphers,
                           hashes = SecioHashes,
-                          pubkey = toHex(localBytesPubkey),
+                          pubkey = localBytesPubkey.shortHexDump,
```

Comment:
`shortHexDump` here is not useful here because in such case you will lose public key type information.

Reply:
.. which can be solved if the key has its own shortlog